examples/sampling/abc-rejection.ipynb

```diff
@@ -0,0 +1,154 @@
+{
```

Comment:
This notebook could do with some more explanation!

- `rejection sampling` --> `[rejection sampling](https://pints.readthedocs.io/en/latest/abc_samplers/rejection_abc_sampler.html)`
- replace the `####` headings with some text explaining what the code block is going to do ("First, we set up a single series problem" etc)
- Add some comments near the log. Is it the "acceptance rate" we're wanting to monitor here?
- Explain the final figure
- Why only one parameter when we've got a 2d problem?